### ISSUE TITLE
maint: failing test  test_ExtractedArchive on maint on NFS

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#4958]
eh heh -- merging of https://github.com/datalad/datalad/pull/4931 (before merging #4953 but after #4949 for which it was green) triggered test failure under NFS:
https://travis-ci.org/github/datalad/datalad/jobs/729943926
```
======================================================================
FAIL: datalad.tests.test_archives.test_ExtractedArchive
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 554, in _wrap_with_tree
    return t(*(arg + (d,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/test_archives.py", line 202, in test_ExtractedArchive
    op.join(fn_archive_obscure, '3.txt')
AssertionError: [' b|;& ΔЙקم๗あ.dbtc / a|;& ΔЙקم๗あ.datc '] != [' b|;& ΔЙקم๗あ.dbtc / a|;& ΔЙקم๗あ.datc ', ' b|;& ΔЙקم๗あ.dbtc /3.txt']
```
when ran in the `maint` branch on `DATALAD_TESTS_SSH=0 _DL_TMPDIR="/tmp/nfsmount" TESTS_TO_PERFORM="datalad.tests datalad.support"` matrix run. 

Let's see if current build after merging #4953 "fixes it"... but I just wanted to make a note since may be some flaky test somehow

**Comment by yarikoptic** [comment]
issue is still persistent, e.g. was present on https://github.com/datalad/datalad/pull/5108

**Comment by yarikoptic** [comment]
ha -- this oldie is a duplicate of a good old #4101 and there also reported seems to be exclusively against maint
